src/server.mjs

```diff
@@ -1001,6 +1038,8 @@ function cleanUpAndSendResult(req, res, next) {
     result.graph = req.graph;
     result.gam = req.withGam === true ? req.gamResults : [];
     result.region = req.region;
+    result.coloredNodes = req.coloredNodes;
+    // add new nodenames field
```

Comment:
```suggestion
```

I think the line this comment describes isn't here? Or is it meant to be about the previous line?